src/main/java/com/tileman/TilemanModeConfig.java

```diff
@@ -78,4 +78,16 @@ default int tilesOffset()
 	{
 		return 20;
 	}
+
+	@Alpha
+	@ConfigItem(
+			keyName = "automarkTiles",
+			name = "Auto-mark tiles",
+			description = "Automatically mark tiles as you walk.",
+			warning = "Will sometimes miss marking a tile"
+	)
+	default boolean automarkTiles()
+	{
+		return false;
```

Comment:
Should this default to true?

Reply:
auto mark defaults to false so new accounts aren't spending their tiles on misclicks